Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayer.h

```diff
@@ -61,7 +61,7 @@ namespace Multiplayer
 
     using ClientMigrationStartEvent = AZ::Event<ClientInputId>;
     using ClientMigrationEndEvent = AZ::Event<>;
-    using ClientDisconnectedEvent = AZ::Event<>;
+    using ClientDisconnectedEvent = AZ::Event<MultiplayerAgentType>;
```

Comment:
Now that this isn't unique to the client, this should probably be renamed EndpointDisonnectedEvent or similar.